### PR TITLE
output cache dir when copying LT/FastText

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: languagetool/.platform.app.yaml [hunk 1]
```diff
@@ -60,13 +61,20 @@ hooks:
     else
       echo "Using cached archive of ${FT} ${FT_VERSION} ..."
     fi
-    echo "Installing archive of ${FT} ${FT_VERSION} ..."
-    cp -R "${PLATFORM_CACHE_DIR}/${FT_DIR}"/* ./"${FT}"
+
+    echo "Installing archive of ${FT} ${FT_VERSION} from '${PLATFORM_CACHE_DIR}/${FT_DIR}' ..."
+    mkdir -p ${FT}
```
Comment (lsmith77) [inline languagetool/.platform.app.yaml:65]: @Szarad this mkdir was missing

File: languagetool/.platform.app.yaml [hunk 2]
```diff
@@ -60,13 +61,20 @@ hooks:
...
+    cp -R "${PLATFORM_CACHE_DIR}/${FT_DIR}"/* "./${FT}"
 
     export FT_MODEL="lid.176.bin"
     if [ ! -f "$FT_MODEL" ]; then
-      echo "Download ${FT_MODEL} ..."
+      echo "Downloading ${FT_MODEL} ..."
```
Comment (lsmith77) [inline languagetool/.platform.app.yaml:70]: added caching here as well

File: languagetool/.platform.app.yaml (hunk 1 above)
Comment (Szarad) [reply languagetool/.platform.app.yaml:65]: mkdir in line 58 wasn't enough?

File: languagetool/.platform.app.yaml (hunk 2 above)
Comment (Szarad) [reply languagetool/.platform.app.yaml:70]: they don't do versioning of this file, but maybe it safer to have it like that as well

Comment (lsmith77) [reply languagetool/.platform.app.yaml:70]: exactly that was my thinking

File: languagetool/.platform.app.yaml (hunk 1 above)
Comment (lsmith77) [reply languagetool/.platform.app.yaml:65]: ah didn't see that one .. the issue is that this was only called while caching but if the cache exists, that mkdir was never reached again

Comment (lsmith77) [reply languagetool/.platform.app.yaml:65]: FYI just pushed to remove the mkdir on line 58

Comment (Szarad) [reply languagetool/.platform.app.yaml:65]: ah I understand now, thanks for the fix

**Review by Szarad — APPROVED** [review]
thanks for changes!

**Pull request merged**